File: src/main/java/com/ject/studytrip/trip/application/dto/TripCategoryInfo.java
package com.ject.studytrip.trip.application.dto;

import com.ject.studytrip.trip.domain.model.TripCategory;

public record TripCategoryInfo(String name, String value) {
    public static TripCategoryInfo from(TripCategory category) {
        return new TripCategoryInfo(category.name(), category.getValue());
    }
}

File: src/main/kotlin/com/ject/studytrip/trip/application/dto/TripCategoryInfo.kt
package com.ject.studytrip.trip.application.dto

import com.ject.studytrip.trip.domain.model.TripCategory

data class TripCategoryInfo(
    val name: String,
    val value: String,
) {
    companion object {
        fun from(category: TripCategory): TripCategoryInfo = TripCategoryInfo(category.name, category.value)
    }
}
